tasks/rsync.yml

```diff
@@ -0,0 +1,33 @@
+---
+- stat:
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  register: stat_current_dir
+
+- name: ANSISTRANO | Remove current folder 
+  file:
+    state: absent
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+#  when: stat_current_dir.stat.islnk is defined and stat_current_dir.stat.islnk
```

Comment:
Why leave this line commented out? I think rollback does not need this step. It is used in deploy so that you can change from symlink to rsync but I think it is not needed here, right?